src/Raven.Studio/typescript/viewmodels/database/settings/saveDatabaseSettingsConfirm.ts

```diff
@@ -0,0 +1,33 @@
+import confirmViewModelBase = require("viewmodels/confirmViewModelBase");
+
+class saveDatabaseSettingsConfirm extends confirmViewModelBase<confirmDialogResult> {
+
+    settingsToSaveText: string;
+    
+    constructor(private databaseSettingsToSave: Array<setttingsItem>) {
+        super();
+        
+        if (this.databaseSettingsToSave.length === 0 ) {
```

Comment:
I wonder if we need this code

Reply:
Code is now refactored w/o the warning text, but I left this condition in as it was more comfortable to generate the relevant html